CHANGELOG.rst

```diff
@@ -51,19 +51,18 @@ TBR
 
           def start_requests(self):
               yield scrapy.Request(
-                  "https://example.com/products/some-product", self.parse
+                  "https://example.com/products/some-product", self.parse_product
               )
 
-          # ✨ NEW: Notice that we're directly using the item here and not the
-          # page object.
-          def parse(self, response: DummyResponse, item: Product):
+          # ✨ NEW: We can directly use the item here instead of the page object.
+          def parse_product(self, response: DummyResponse, item: Product) -> Product:
```

Comment:
Now that we have NO_CALLBACK, I wonder if we should promote `parse` again, even if it would break with Scrapy 2.7 and earlier.

Reply:
I thought about this but I went the safe route for now since not all users have the luxury of updating immediately to Scrapy 2.8 right away. Perhaps we can use `parse` again after some time.